docs/reference/models/transformers_multimodal.md

```diff
@@ -0,0 +1,196 @@
+# Transformers Multimodal
+
+Outlines allows seamless use of [multimodal models](https://huggingface.co/learn/multimodal-models-course/en/unit1/introduction).
+
+`outlines.models.TransformersMultimodal` shares interfaces with, and is based on [outlines.models.Transformers](./transformers.md).
+
+## Create a `TransformersMultimodal` model
+
+`TransformersMultimodal` models inherit from `Transformers` and accept similar initialization parameters, with the exception that you should provide a processor instead of a tokenizer.
+
+You can use `outlines.from_transformers` to load a `transformers` model and processor:
+
+```python
+from transformers import CLIPModel, CLIPProcessor
+from outlines import models
+
+model_name = "openai/clip-vit-base-patch32"
+model = models.from_transformers(
+    CLIPModel.from_pretrained(model_name),
+    CLIPProcessor.from_pretrained(model_name)
+)
+```
+
+You must provider a processor adapted to your model. We currently only support vision and audio models.
+
+## Use the model to generate text from prompts and images/audios
+
+When calling the model, the prompt argument you provide must be a dictionary using the following format:
+```
+{
+    "text": Union[str, List[str]],
+    "images": Optional[Union[Any, List[Any]]],
+    "audios": Optional[Union[Any, List[Any]]],
```

Comment:
```suggestion
    "audio_files": Optional[Union[Any, List[Any]]], 
```

Reply:
I used `audios` because it's the keyword used by the Gwen2Audio processor (for the same reason I changed `prompts` into `text`). I think letting users choose whatever keyword their processor uses (apart from text that's shared by all) may be the best solution. We would still use `inputs = self.processor(**model_input, padding=True, return_tensors="pt")`. Another advantage is that it lets users provide optional kwargs for the processor call.

Reply:
If those are the name of the arguments in `transformers` so we can do `inputs = self.processor(**model_input, padding=True, return_tensors="pt")` let's keep them this way.